packages/material-ui/src/MenuItem/MenuItem.d.ts

```diff
@@ -10,6 +11,43 @@ export type MenuItemTypeMap<P = {}, D extends React.ElementType = 'li'> = Omit<
   'classKey'
 > & {
   classKey: MenuItemClassKey;
+  props: P & {
+    /**
+     * Fires when the right arrow key is pressed
+     * on a MenuItem that contains nested items
+     * and passes focus to the first child of the
+     * nested items array
+     */
+    handleArrowRightKeydown?: React.ReactEventHandler<{}>;
+    /**
+     * An array of MenuItems to render in a nested Menu
+     */
+    nestedItems?: React.ReactNode;
```

Comment:
This should be an array of components. I think the syntax looks like this (but not 100% sure):

```suggestion
    nestedItems?: React.ReactNode[];
```

Reply:
Not sure about that, Menu's definition for children is `children?: React.ReactNode;`

Reply:
https://flow.org/en/docs/react/types/#toc-react-node

Reply:
Huh. Interesting.